tasks/setup.yaml

```diff
@@ -33,14 +33,20 @@ tasks:
           else
             ./uds deploy oci://defenseunicorns/uds/${{ if eq .inputs.snapshots "true" }}snapshots/${{ end }}bundles/k3d-core-slim-dev:${{ .inputs.version }} --set INSECURE_ADMIN_PASSWORD_GENERATION=${{ .inputs.insecure_keycloak_admin }} --confirm --no-progress ${{ .inputs.options }}
           fi
+      - description: Validate Keycloak endpoint
+        wait:
+          network:
+            protocol: https
+            address: sso.uds.dev
+            code: 200
```

Comment:
Just want to callout we can add this upstream in the checkpoint package itself if this seems to help (probably in the next release). Wanted to ship it here first to test though - based on what we were seeing it seems like some timing inconsistencies around Istio configuring all the right routing and rbac when the checkpoint is "unfrozen". Waiting on keycloak accessibility through the VS (rather than just pod readiness) seems to help.